core/platform/auth/firebase_auth_services.py

```diff
@@ -127,6 +127,32 @@ def authenticate_request(request):
         return claims.get('sub', None)
 
 
+def delete_user(user_id):
+    """Deletes all associations referring to the given user_id.
+
+    Args:
+        user_id: str. The ID of the user being deleted.
+
+    Returns:
+        bool. Whether the user's data has been successfully deleted.
+    """
+    mapping = auth_models.UserIdByFirebaseAuthIdModel.get_by_user_id(user_id)
```

Comment:
No need to delete this model manually, since its `DELETION_POLICY` is `DELETE_AT_END` it will be deleted after we confirm that the deletion for all other models went fine (`wipeout_service._delete_user_models_with_delete_at_end_policy`).

Reply:
Done.